skins/cat17/src/scripts/donationForm.js

```diff
@@ -162,10 +162,6 @@ $( function () {
         viewHandler: WMDE.View.createFieldValueValidityIndicator( $('#post-code-company') ),
         stateKey: 'donationInputValidation.postcode'
       },
-      {
-        viewHandler: WMDE.View.createFieldValueValidityIndicator( $( '#post-code' ) ),
-        stateKey: 'donationInputValidation.postcode'
-      },
```

Comment:
This was unused?

Reply:
Yes an no. There is a second identical call on l158 which should suffice.